objects/o_player/Create_0.gml

```diff
@@ -6,7 +6,7 @@ jumpspd		= -8;
 maxfallspd	= 6;
 
 function on_ground() {
-	return (place_meeting(x, y+1, o_solid));
+	return (place_meeting(x, y+1, o_solid) || place_meeting(x, y+1, o_solid_oneway));
```

Comment:
we should make an `on_solid()` function in the script I mentioned. It would essentially do this one line, and then we can add more parents to it in the future. No need to make the user figure that out themselves

Reply:
Agreed. We'll make a global script to include some shared functions.